c_glib/gandiva-glib/expression.cpp

```diff
@@ -110,31 +111,23 @@ ggandiva_expression_class_init(GGandivaExpressionClass *klass)
 
 /**
  * ggandiva_expression_new:
- * @function: The function name in the expression.
- * @input_fields: (element-type GArrowField): The input fields.
- * @output_field: A #GArrowField to be output.
+ * @node: The node in the expression tree.
+ * @field: A #GArrowField to be output.
```

Comment:
Can you use `root_node` and `result_field` here?
Parameter name is important to describe parameter value.

Reply:
I've renamed it.